x-pack/plugins/ml/public/application/trained_models/models_management/test_models/models/lang_ident/lang_ident.tsx

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as estypes from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import React, { FC, useState, useMemo } from 'react';
+
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import { EuiSpacer, EuiTextArea, EuiButton, EuiTabs, EuiTab } from '@elastic/eui';
+
+import { useMlApiContext } from '../../../../../contexts/kibana';
+import { LangIdentInference } from './lang_ident_inference';
+import type { FormattedLangIdentResp } from './lang_ident_inference';
+import { LangIdentOutput } from './lang_ident_output';
+import { MLJobEditor } from '../../../../../jobs/jobs_list/components/ml_job_editor';
+import { extractErrorMessage } from '../../../../../../../common/util/errors';
+import { ErrorMessage } from '../../inference_error';
+import { OutputLoadingContent } from '../../output_loading';
+
+interface Props {
+  model: estypes.MlTrainedModelConfig;
+}
+
+enum TAB {
+  TEXT,
+  RAW,
+}
+
+export const LangIdentModel: FC<Props> = ({ model }) => {
+  const { trainedModels } = useMlApiContext();
+
+  const ner = useMemo(() => new LangIdentInference(trainedModels, model), [trainedModels, model]);
+
+  // const [inputText, setInputText] = useState('');
+  const [inputText, setInputText] =
+    useState(`Hola, mi nombre es James y vivo en Hersham, Surrey con mi esposa Kate y mis hijos Jonathan y Emily.
+
+  Solía ​​vivir en Londres, pero nos mudamos a los palos hace unos 3 años.`);
+  const [isRunning, setIsRunning] = useState(false);
+  const [output, setOutput] = useState<FormattedLangIdentResp | null>(null);
+  const [rawOutput, setRawOutput] = useState<string | null>(null);
+  const [selectedTab, setSelectedTab] = useState(TAB.TEXT);
+  const [showOutput, setShowOutput] = useState(false);
+  const [errorText, setErrorText] = useState<string | null>(null);
+
+  async function run() {
+    setShowOutput(true);
+    setOutput(null);
+    setRawOutput(null);
+    setIsRunning(true);
+    setErrorText(null);
+    try {
+      const { response, rawResponse } = await ner.infer(inputText);
+      setOutput(response);
+      setRawOutput(JSON.stringify(rawResponse, null, 2));
+    } catch (e) {
+      setIsRunning(false);
+      setOutput(null);
+      setErrorText(extractErrorMessage(e));
+      setRawOutput(JSON.stringify(e.body ?? e, null, 2));
+    }
+    setIsRunning(false);
+  }
+
+  return (
+    <>
+      <EuiTextArea
+        placeholder="Input text"
+        value={inputText}
+        disabled={isRunning === true}
+        fullWidth={true}
```

Comment:
```suggestion
        fullWidth
```

Reply:
Updated in https://github.com/elastic/kibana/pull/128359/commits/2a9f47840a7e077a838e19e46673fad819b0f80c